scripts/get-nightly-gitlab.py

```diff
@@ -32,7 +32,7 @@ def iterate_dates(start_date, total_days, delta_days=-1):
 
 def iterate_nightlies(start_date, total_days):
     # Filter with user that triggers master and nightlies:
-    parameters_base = "username=lluiscampos"
+    parameters_base = "username=mender-test-bot"
 
```

Comment:
Additionally,

With the intention of experimenting a bit with GCP resources, I yesterday moved the nightly to start at 10pm, so the filter few lines down is also off :see_no_evil:

Reply:
Oh.. hah. Does it show as `updated` when it's finished? :thinking:

Reply:
I don't know what you are asking...

In any case, I restored schedule to 1am so you can ignore my comment.